### ISSUE TITLE
unable to update anexistign static url

### ISSUE THREAD

**Issue by lemmy04** [cydrobolt/polr#537]
## Expected Behavior
I'm trying to use polr on my own server to create a static URL that points to a dynamic one that changes all the time. For that purpose I have a script on the device star serves the dynamic url which uses wget to create an entry in my polr server that has a completely static url - by using the custom_ending and is_public=true.

My problem is, it only works once - after that I get "custom ending already in use" until I manually log in on the dashboard (as admin, even) to delete the entry.

## Current Behavior
it is impossible to update an existing url

## Possible Solution
implement a new parameter, "update=<bool>", or implement a delete action in the API.

## Steps to Reproduce (for bugs)
1. create an URL via the API
2. create the same url with a different target
3. observe the second attempt to fail.

## Context
Basically, the way this affects me is: polr is useless for my project.

## Your Environment
latest release from github running on openSUSE 15.1 with PHP7
